API-proposed.yaml

```diff
@@ -2209,6 +2209,8 @@ components:
           type: integer
           description: End time for this state
           format: int64
+      description: "If state holds a single applicable value and no style(s), that\
```

Comment:
This is not related to series model styles and should not be part of this PR.

Reply:
Indeed, as PR's description above. We already used such PRs and strived for focused (related) commits underneath, as in here. But I can work on splitting it into a few more branches.

Reply:
Correction, this commit is not focused enough yes, so I'm currently splitting it (this PR) into focused ones. Thanks for the flag.

Reply:
Done; split into this now focused main PR commit, alongside the new #80 and #81 ones.